Core/src/Surfaces/AnnulusBounds.cpp

```diff
@@ -389,7 +389,7 @@ bool AnnulusBounds::inside(const Vector2& lposition,
     // Calculate minDist based on weight from the boundary tolerance object.
     // That weight matrix is in STRIP PC
     auto [delta, minDist] =
-        closestPointDistanceBound(boundaryToleranceChi2.weight);
+        closestPointDistanceBound(boundaryToleranceChi2.weightMatrix());
```

Comment:
Looks unrelated ? But the PR is small - so it's ok to get the change to Eigen::Map in also.